collect_app/src/androidTest/java/org/odk/collect/android/regression/AdminSettingsTest.java

```diff
@@ -1,20 +1,38 @@
 package org.odk.collect.android.regression;
 
+import android.Manifest;
+
+import androidx.test.rule.GrantPermissionRule;
 import androidx.test.runner.AndroidJUnit4;
 
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.RuleChain;
 import org.junit.runner.RunWith;
+import org.odk.collect.android.support.CollectTestRule;
+import org.odk.collect.android.support.ResetStateRule;
 import org.odk.collect.android.support.pages.AdminSettingsPage;
-import org.odk.collect.android.support.pages.MainMenuPage;
 
 //Issue NODK-239
 @RunWith(AndroidJUnit4.class)
-public class AdminSettingsTest extends BaseRegressionTest {
+public class AdminSettingsTest {
+
+    public CollectTestRule rule = new CollectTestRule();
+
+    @Rule
+    public RuleChain copyFormChain = RuleChain
+            .outerRule(GrantPermissionRule.grant(
+                    Manifest.permission.READ_EXTERNAL_STORAGE,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE,
+                    Manifest.permission.READ_PHONE_STATE
+            ))
+            .around(new ResetStateRule())
+            .around(rule);
```

Comment:
This test needed permissions but as we've seen before it's important to chain the rules so permissions are definitely granted **before** any Actvities launch.